index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <script src="https://kit.fontawesome.com/9dfa895fe8.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div class="content">
+    <div class="navbar">
+      <div class="logo">
+        <h2>My Logo</h2>
+      </div>
+      <div class="icon">
+        <img src="images/Union.jpg" alt="Union">
+      </div>
+    </div>
+    <div class="section">
+      <img class="img1" src="images/Ellipse11.jpg" alt="Ellipse11">
+      <img class="img2" src="images/Ellipse12.svg" alt="Ellipse12">
+      <img class="img3" src="images/Ellipse8.svg" alt="Ellipse8">
+      <img class="img4" src="images/Ellipse4.svg" alt="Ellipse4">
+      <img class="img5" src="images/Ellipse5.svg" alt="Ellipse5">
+      <img class="img6" src="images/Rectangle7.svg" alt="Rectangle7">
+      <img class="img7" src="images/Ellipse3.jpg" alt="Ellipse 3">
```

Comment:
- [ ] The decorations are always background images, not `img` tags. Let's add the background images to the `css` file and remove them from the `html` file. Check out these links to learn how to do that: [link 1](https://www.w3schools.com/cssref/pr_background-image.asp), [link 2](https://developer.mozilla.org/en-US/docs/Web/CSS/background-image)